### PR TITLE
Add `feedbackSurveyCompleted` event to Customer Center events

### PR DESCRIPTION
**Pull request by vegaro**
Sends a `feedbackSurveyCompleted` event when an option in the feedback survey is pressed

### CODE REVIEW

File: RevenueCat.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -6547,7 +6547,7 @@
 				SUPPORTED_PLATFORMS = "appletvos appletvsimulator iphoneos iphonesimulator macosx watchos watchsimulator xros xrsimulator";
 				SUPPORTS_MACCATALYST = YES;
 				SUPPORTS_XR_DESIGNED_FOR_IPHONE_IPAD = YES;
-				SWIFT_ACTIVE_COMPILATION_CONDITIONS = DEBUG;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = "DEBUG CUSTOMER_CENTER_ENABLED";
```
Comment (jamesrb1) [inline RevenueCat.xcodeproj/project.pbxproj:6550]: Oops?

File: RevenueCat.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -6896,6 +6896,7 @@
 				SDKROOT = iphoneos;
 				SKIP_INSTALL = NO;
 				SUPPORTS_MACCATALYST = YES;
+				SWIFT_ACTIVE_COMPILATION_CONDITIONS = CUSTOMER_CENTER_ENABLED;
```
Comment (jamesrb1) [inline RevenueCat.xcodeproj/project.pbxproj:6899]: Also oops?

**Review by jamesrb1 — COMMENTED** [review]
Looks good other than enabling customer center.

File: RevenueCat.xcodeproj/project.pbxproj (hunk 1 above)
Comment (vegaro) [reply RevenueCat.xcodeproj/project.pbxproj:6550]: Oops yes

**Pull request merged**